apps/services/handlers.py

```diff
@@ -270,5 +283,10 @@ async def __generate_user_document_name(cls, service_id: int, session: AsyncSess
         )
         return f"Заява на {service.service_name.lower()}"
 
+    @classmethod
+    async def __generate_user_document_name_for_download(cls, document_name: str, user_id: int, session: AsyncSession) -> str:
+        student = await student_service.read(session=session, data={"user_id": user_id})
+        return f"{document_name.replace(' ', '_')}_{student.first_name}_{student.last_name}.docx"
```

Comment:
f-string is better than str + str construction

Reply:
Ok